test/unit/tokenizer_unit_test.rb

```diff
@@ -24,15 +24,24 @@ def test_tokenize_blocks
   def test_calculate_line_numbers_per_token_with_profiling
     template = Liquid::Template.parse("", :profile => true)
 
-    assert_equal [1],       template.send(:tokenize, "{{funk}}").map(&:line_number)
-    assert_equal [1, 1, 1], template.send(:tokenize, " {{funk}} ").map(&:line_number)
-    assert_equal [1, 2, 2], template.send(:tokenize, "\n{{funk}}\n").map(&:line_number)
-    assert_equal [1, 1, 3], template.send(:tokenize, " {{\n funk \n}} ").map(&:line_number)
+    assert_equal [1],       to_a(template.send(:tokenize, "{{funk}}")).map(&:line_number)
```

Comment:
Yup, the `tokenize` helper just needs an options hash, then the `line_numbers: true` option can be passed into the tokenizer.